### ISSUE TITLE
PHP TWIG Support

### ISSUE THREAD

**Issue by webdevilopers** [rubychan/coderay#197]
We are using CodeRay for Redmine. Since CodeRay already supports PHP maybe it could also support TIWG - somewhat an equivalent to HAML.

**Comment by korny** [comment]
Yes! If you want to try, here's a [tutorial on writing scanners](https://github.com/rubychan/coderay/wiki/Scanner-Requests). You also need some test code for https://github.com/rubychan/coderay-scanner-tests.

**Comment by webdevilopers** [comment]
Unfort. my experience w/ Ruby and expressions aren't good enough to write the scanner myself.
But I think some of my PHP TWIG friends are also experienced in Ruby. Maybe I can find somebody who is using Redmine too.
